src/java/org/apache/cassandra/db/compaction/unified/AdaptiveController.java

```diff
@@ -297,6 +297,11 @@ private static int[] readStoredScalingParameters(JSONArray storedScalingParamete
         }
     }
 
+    protected void setScalingParameters(String newScalingParameters)
+    {
+        scalingParameters = parseScalingParameters(newScalingParameters);
+    }
+
```

Comment:
can you create a corresponding CNDB PR with some integration tests to verify new setting is applied when vector index is created,